ui/package.json

```diff
@@ -114,22 +116,20 @@
     "karma-webdriver-launcher": "^1.0.5",
     "lolex": "^3.0.0",
     "prettier": "^2.0.4",
-    "react-app-rewired": "2.1.8",
-    "react-scripts": "4.0.3",
+    "react-app-rewired": "^2.1.9",
```

Comment:
resolves an issue which didn't appear for us under react-scripts 4: https://github.com/timarney/react-app-rewired/issues/585